src/modules/sign/components/Signin.jsx

```diff
@@ -24,7 +24,7 @@ const SignIn = () => {
 			</div>
 
 			<LogInContainer />
-			<p className='bottom_text'>Trusted by Tech Companies around the world</p>
+			<p style={{fontFamily: "sans-serif"}} className='bottom_text'>Trusted by Tech Companies around the world</p>
```

Comment:
Please move the inline style to the `bottom_text` class

Reply:
Roger that.